core/src/navigation/LeftNav.html

```diff
@@ -201,7 +224,11 @@
           <!-- Category nodes -->
           <li
             class="fd-nested-list__group-header lui-category"
-            title="{$getTranslation(key)}"
+            title="{
+              (nodes.tooltipText && nodes.tooltipText.length) ?
+              getTooltipText(nodes) :
```

Comment:
I find a flaw in this logic. 
```
if ( nodes.tooltipText && nodes.tooltipText.length ) {
    //nodes.tooltipText is definitely not false or empty here
    // the following 'getTooltipText' might never return 'defaultsTooltipTextConfig' as 'node.tooltipTex' will initially always be true, never reaching second one.
     getTooltipText(nodes) => return node.tooltipText || defaultsTooltipTextConfig; =
} else {
// (nodes.tooltipText === false || nodes.tooltipText === '') ? '' : $getTranslation(key)
}
```
Due to that I also don't get the **defaults.tooltipText** affecting the tooltips at all if I add it in the luigi configurations, while testing locally. I believe this might be the reason 🤔 . Does it work for you ?

This would be made much easier if **defaults.tooltipText** was documented, so I could also know what is the intended result.

Reply:
You have right. I changed logic.

`function resolveTooltipText(node, translation) {
    if (node.tooltipText && (node.tooltipText === undefined || node.tooltipText === null)) {
      return defaultsTooltipTextConfig;
    }

    if (node.tooltipText && node.tooltipText.length) {
      return node.tooltipText;
    }

    if (node.tooltipText === false || node.tooltipText === '') {
      return '';
    }

    return translation;
  }`